webpack.config.js

```diff
@@ -8,7 +9,8 @@ const ProductionPlugins = [
   // production webpack plugins go here
   new webpack.DefinePlugin({
     "process.env": {
-      NODE_ENV: JSON.stringify("production")
+      NODE_ENV: JSON.stringify("production"),
+      REACT_APP_AXIOS_HOST: JSON.stringify(process.env.REACT_APP_AXIOS_HOST),
```

Comment:
this didn't actually work, and i'll revert the change later.  current plan is to avoid messing w/ env in react until we (potentially) switch to Vite or another solution presents itself

Reply:
reverted